src/rpcrawtransaction.cpp

```diff
@@ -808,9 +808,9 @@ UniValue sendrawtransaction(const UniValue& params, bool fHelp)
         throw JSONRPCError(RPC_DESERIALIZATION_ERROR, "TX decode failed");
     uint256 hashTx = tx.GetHash();
 
-    bool fOverrideFees = false;
-    if (params.size() > 1)
-        fOverrideFees = params[1].get_bool();
+    CAmount nMaxRawTxFee = maxTxFee;
```

Comment:
Yes, maybe even replacing the boolean (although it breaks the old API). instead of override=true, people could use maxfee=0.